recipes-core/images/host-toolchain-sysroot.bb

```diff
@@ -60,8 +60,6 @@ create_sdk_files_append() {
     bbnote "Target Sysroot: Remove escaping symlinks"
     rm -Rf "${SDK_OUTPUT}/${SDKTARGETSYSROOT}/etc/mtab"
     rm -Rf "${SDK_OUTPUT}/${SDKTARGETSYSROOT}/var/lock"
-    rm -Rf "${SDK_OUTPUT}/${SDKTARGETSYSROOT}/usr/lib/lsb/install_initd"
-    rm -Rf "${SDK_OUTPUT}/${SDKTARGETSYSROOT}/usr/lib/lsb/remove_initd"
```

Comment:
Could you split this patchset into two commits: one which removes the deprecated LSB references, and one which adds the lsbinitscripts recipe to meta-nilrt.